docs/DeveloperGuide.md

```diff
@@ -257,13 +257,15 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
-
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+* An artist who works through commissions
+* has a need to manage clients and commissions
+* wants to review past commission works and feedbacks
+* wants to market himself to the right clients
+* prefers desktop apps over other types
+* prefers using GUI over CLI
```

Comment:
For v1.2 graphic interface won't be implemented, so maybe keep this as 

- is reasonably comfortable using CLI apps

Reply:
We also have to adhere to the non-functional requirement for catering to users who prefer typing: https://nus-cs2103-ay2223s1.github.io/website/admin/tp-constraints.html#constraint-typing-preferred. Perhaps we can bring back the original line on preferring typing to fulfil this requirement.

Reply:
Ok got it